react/react1/week3/homework/src/BorderComponent.js

```diff
@@ -0,0 +1,9 @@
+import './App.css';
+
+export default function BorderComponent(props) {
+    return (
+        <div className={"borderComponent-" + props.color}>
```

Comment:
Here is the issue. This prop is working but not the way you may have originally intended.